view.go

```diff
@@ -522,12 +672,75 @@ func (v *View) Word(x, y int) (string, error) {
 	return string(str[nl:nr]), nil
 }
 
+// Hide view
+func (v *View) Hide() {
```

Comment:
We could also just expose the `visible` attribute, That would make it easier to use for the end user and cleanup some code.

Reply:
We should be careful with this, what if some time later, one will need to execute some code (such as cleanup, or what ever), when user hides / shows the view.

I'm ok with current solution. If you don't like it, just tell, I can make a commit, which will expose the `visible` attribute.

Reply:
The current solutions works fine but do there really need to be 4 functions for setting and viewing 1 Boolean? (These functions will also show up in the user there code editor in staid of 1 attribute)  

Beside that exposing just the `Visible` attribute does make code writing easier,  
For example when the user already has the value:  
```go
// In staid of writing out a if else statement
if someValue {
  v.Show()
} else {
  v.Hide()
}

// It can be this way
v.Visible = someValue
```
Also something that is maybe overlooked but `Visible` is easier to remember than `Show`, `Hide`, `ToggleVisible` and `Visible`

I see what you mean but we can change it always later if we want to.  
This will definitely cause issues later for people that want to upgrade but from my experience I've had lots of projects with critical code changes over time that required me to re-base my code, that's why projects should use a vendor folder.

Reply:
Exposed `Visible` in [7799456](https://github.com/awesome-gocui/gocui/pull/24/commits/7799456673acc0b02f742c4aa079edc4e598b59b).